Hotfix airpollution standards

gabriel-abrahao opened the pull request:
## Purpose of this PR

Changes standard `cm_APscen` to CLE and updating scenario_configs

## Type of change

*Indicate the items relevant for your PR by replacing* :white_medium_square: *with* :ballot_box_with_check:.\
*Do not delete any lines. This makes it easier to understand which areas are affected by your changes and which are not.*

### Parts concerned
- :white_medium_square: GAMS Code
- :white_medium_square: R-scripts
- :white_medium_square: Documentation (GAMS incode documentation, comments, tutorials)
- :white_medium_square: Input data / CES parameters
- :white_medium_square: Tests, CI/CD (continuous integration/deployment)
- :white_medium_square: Configuration (switches in main.gms, default.cfg, and scenario_config*.csv files)
- :white_medium_square: Other (please give a description)

### Impact
- :white_medium_square: Bug fix
- :white_medium_square: Refactoring
- :white_medium_square: New feature
- :white_medium_square: Change of parameter values or input data (including CES parameters)
- :white_medium_square: Minor change (default scenarios show only small differences)
- :white_medium_square: Fundamental change of results of default scenarios

## Checklist

*Do not delete any line. Leave **unfinished** elements unchecked so others know how far along you are.\
In the end all checkboxes must be ticked before you can merge*.

- [ ] **I executed the automated model tests (`make test`) after my final commit and all tests pass (`FAIL 0`)**
- [ ] **I adjusted the reporting in [`remind2`](https://github.com/pik-piam/remind2) if and where it was needed**
- [ ] **I adjusted the madrat packages (mrremind and other packages involved) for input data generation if and where it was needed**
- [ ] My code follows the [coding etiquette](https://github.com/remindmodel/remind/blob/develop/main.gms#L80)
- [ ] I explained my changes within the PR, particularly in hard-to-understand areas
- [ ] I checked that the [in-code documentation](https://github.com/remindmodel/remind/blob/develop/main.gms#L120) is up-to-date
- [ ] I adjusted `forbiddenColumnNames` in [readCheckScenarioConfig.R](https://github.com/remindmodel/remind/blob/develop/scripts/start/readCheckScenarioConfig.R) in case the PR leads to deprecated switches
- [ ] I updated the `CHANGELOG.md` [correctly](https://gitlab.pik-potsdam.de/rse/rsewiki/-/wikis/Standards-for-Writing-a-Changelog) (added, changed, fixed, removed, input data/calibration)

## Further information (optional)

* Runs with these changes are here:
* Comparison of results (what changes by this PR?):

robertpietzcker reviewed (approved):
thanks, Gabriel!

once you change the main.gms, I would say ready to merge

gabriel-abrahao commented:
Thanks @dklein-pik . I don't have the time right now to do that on all of them, but I cleaned the standard `scenario_config.csv` and made sure all others make sense. I also changed the logic of the AP scenario choice based on a discussion with Laurin, and now they are following more the ScenarioMIP logic.

gabriel-abrahao commented on main.gms:
```diff
@@ -1885,7 +1885,7 @@ $setGlobal cm_APssp  FROMGDPSSP          !! def = SSP2
 *' (VLE): Very strong Legislation Emissions
 *' (MFR): Maximum Feasible Reductions
-$setGlobal cm_APscen  SSP2          !! def = SSP2
+$setGlobal cm_APscen  CLE          !! def = SSP2
```
Ops, I see, done!
